db/migrate/20210301172205_create_topics.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :topics do |t|
+      t.string :name, :null => false
```

Comment:
we usually prefer this syntax
```suggestion
      t.string :name, null: false
```